graphql_compiler/tests/test_data_tools/integration_data/create_orientdb_integration.sql

```diff
@@ -3,3 +3,6 @@ create vertex Animal set name = 'Animal 1', net_worth = Decimal('100'), uuid = '
 create vertex Animal set name = 'Animal 2', net_worth = Decimal('200'), uuid = 'cfc6e625-8594-0927-468f-f53d864a7a52'
 create vertex Animal set name = 'Animal 3', net_worth = Decimal('300'), uuid = 'cfc6e625-8594-0927-468f-f53d864a7a53'
 create vertex Animal set name = 'Animal 4', net_worth = Decimal('400'), uuid = 'cfc6e625-8594-0927-468f-f53d864a7a54'
+create vertex Species set name = 'Species 1', uuid = 'ce4f0889-ecdb-4e27-8ffa-3140eb507549', alias = ['Species 1']
+create vertex Species set name = 'Species 2', uuid = '660b4d07-37fe-4eeb-bb9f-fbc3845e35a9', alias = ['Species 2']
+create edge Entity_Related from (select from Species where name = 'Species 1') to (select from Species where name = 'Species 2')
```

Comment:
This doesn't seem relevant?

Reply:
The query asks for animals, and gets back species. This is just the data needed to expose the bug.

Reply:
Maybe make the aliases be something that is not equal to the `name` field? Maybe add an underscore instead of a space, or something like that.

Reply:
Changed the test to use a runtime parameter instead of a colocated tagged parameter. It is simpler, removes the need for aliases, and does what I need.